riverdriver/riverdatabasesql/river_database_sql_driver.go

```diff
@@ -999,6 +994,18 @@ func (w templateReplaceWrapper) QueryRowContext(ctx context.Context, sql string,
 	return w.dbtx.QueryRowContext(ctx, sql, args...)
 }
 
+// lib/pq reads in `bits` (like `bits(8)`) as a bit string that's a normal
+// decimal integer like 1111_1111 (_not_ 0b1111_1111). This function converts it
+// back to binary representation. See the test suite for examples.
```

Comment:
pq 🤦‍♂️

Reply:
Yeah haha. There might be some other way around this, but at least this isn't _that_ bad.